plugins/modules/gitlab_group_variable.py

```diff
@@ -15,7 +15,8 @@
 version_added: 1.2.0
 description:
   - Creates a group variable if it does not exist.
-  - When a group variable does exist, its value is updated when the values are different.
+  - When a group variable does exist and is not hidden, its value is updated when the values are different.
+  - When a group variable does exist and is hidden, its value is updated. In this case, the module is B(not idempotent).
```

Comment:
I think these should be in the same paragraph.
```suggestion
    When a group variable does exist and is hidden, its value is updated. In this case, the module is B(not idempotent).
```